app/services/BookingService.js

```diff
@@ -24,5 +25,17 @@ class BookingService {
       data: bookedTrip,
     })
   }
+
+  static async getAll(req, res) {
+    const bookings = await Booking.getAllBookings()
+    const formattedBookings = await Promise.all(
+      bookings.map(booking => formatBooking(booking)),
+    )
+    return res.status(200).send({
+      status: 200,
+      message: 'Successfully',
```

Comment:
Provide a more descriptive message here

Reply:
Thank you for the feedback. I am working on it.